src/Car.java

```diff
@@ -0,0 +1,14 @@
+public class Car {
+
+    public void start() {
+        startElectricity();
+        startCommand();
+        startFuelSystem();
```

Comment:
Let's make it a bit more interesting. Please add a print statement at the end of the method's `start()` body - "Engine is started". Add the respective print statements for each method that is called within the method `start()`. For example, "Electricity is started" etc.